specification/compute/resource-manager/Microsoft.Compute/DiskRP/stable/2022-07-02/snapshot.json

```diff
@@ -623,6 +627,10 @@
           "$ref": "./diskRPCommon.json#/definitions/EncryptionSettingsCollection",
           "description": "Encryption settings collection used be Azure Disk Encryption, can contain multiple encryption settings per disk or snapshot."
         },
+        "incrementalSnapshotFamilyId": {
```

Comment:
This property cannot be set by the client. Should not be there in SnapshotUpdateProperties

Reply:
Removed